README.md

```diff
@@ -568,6 +568,35 @@ The former can be ameliorated by opening multiple traces; the latter
 can be fixed by rearchitecting lager's file backend, but this has not
 been tackled.
 
+### Traces from configuration
```

Comment:
Since one of your examples (`event.log`) won't work with any existing release of lager, please add a note that the log level is required for the file backend in traces up to and including <fill-in-existing-2.x/3.x releases>.